flict/__main__.py

```diff
@@ -151,18 +149,23 @@ def parse():
     parser_v = subparsers.add_parser(
         'verify', help='verify license compatibility')
     parser_v.set_defaults(which="verify", func=verify)
+    parser_v.add_argument('--outbound-license', '-ol', type=str, dest='out_license', help='Outbound license', default=None)
+    parser_v.add_argument('--inbound-license', '-il', type=str, nargs='+', dest='in_license_expr', help='Inbound license(s)', default=[])
+    parser_v.add_argument('--sbom', '-s', type=str, dest='verify_sbom', help='')
```

Comment:
I'd prefer if you either remove the help options or fill it with something meaningful